pyaedt/edb_core/siwave.py

```diff
@@ -419,48 +691,48 @@ def create_circuit_port(self, positive_component_name, positive_net_name, negati
             Name of the negative component. The default is ``None``, in which case the name of
             the positive net is assigned.
         negative_net_name : str, optional
-            Name of the negative net name. The default is ``"GND"``.
+            Name of the negative net name. The default is ``None`` which will look for *GND Nets.
```

Comment:
Sphinx won't like `*`.  We can either escape it or make the docstring raw with `r`
```suggestion
            Name of the negative net name. The default is ``None`` which will look for \*GND Nets.
```

Reply:
good to know